pkg/pgsql/server/bmessages/data_row.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2021 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bmessages
+
+import (
+	"bytes"
+	"encoding/binary"
+	"github.com/codenotary/immudb/pkg/api/schema"
+)
+
+func DataRow(rows []*schema.Row, colNumb int, binaryFormat bool) []byte {
+	rowsB := make([]byte, 0)
+	for _, row := range rows {
+		rowB := make([]byte, 0)
+		////##-> dataRow
```

Comment:
Just out of curiosity: this comment seems to follow some unclear convention - e.g. being a kind of start marker for a block of code? But then i don't see an end marker for the block (e.g. i was expecting something like `////##<-` somewhere lower) ...